pkg/runtime/scheduler/streamer.go

```diff
@@ -157,10 +162,12 @@ func (s *streamer) invokeApp(ctx context.Context, job *schedulerv1pb.WatchJobsRe
 	switch codes.Code(statusCode) {
 	case codes.OK:
 		log.Debugf("Sent job %s to app", job.GetName())
+		diag.DefaultComponentMonitoring.JobExecutedSuccess(ctx, job.GetName(), diag.JobExecutionOp, true, elapsed)
 	case codes.NotFound:
 		log.Errorf("non-retriable error returned from app while processing triggered job %s. status code returned: %v", job.GetName(), statusCode)
 	default:
 		log.Errorf("unexpected status code returned from app while processing triggered job %s. status code returned: %v", job.GetName(), statusCode)
+		diag.DefaultComponentMonitoring.JobExecutedFailure(ctx, job.GetName(), diag.JobExecutionOp, true, elapsed)
```

Comment:
I think this should be false since its used as a success bool in the func. Currently, for both a success & failure scenario this is being set to true.

```suggestion
		diag.DefaultComponentMonitoring.JobExecutedFailure(ctx, job.GetName(), diag.JobExecutionOp, false, elapsed)
```